app/javascript/application.js

```diff
@@ -1,3 +1,16 @@
 // Configure your import map in config/importmap.rb. Read more: https://github.com/rails/importmap-rails
 import "@hotwired/turbo-rails"
 import "controllers"
+
+// window.addEventListener("turbo:load", () => {
+//   hideNotice();
+// });
+
+// function hideNotice() {
+//   const notification = document.querySelector('.notification')
+//   if (notification) {
+//     setInterval(function() {
+//       notification.classList.add('fade');
+//     }, 2000);
+//   }
+// }
```

Comment:
- [OPTIONAL] Please do not leave unused code as a comment in your codebase. this will make your code much noisier, It is always a good practice to keep your code clean from such comments. 😉 👍🏻

Reply:
Hi @BenMukebo, you are right I forget that pice of code at all...